demos/src/demo-stacked-bar.js

```diff
@@ -132,15 +142,15 @@ function createHorizontalStackedBarChart(optionalColorSchema) {
 if (d3Selection.select('.js-stacked-bar-chart-tooltip-container').node()){
     // Chart creation
     createStackedBarChartWithTooltip();
-    createHorizontalStackedBarChart();
+    // createHorizontalStackedBarChart();
 
     // For getting a responsive behavior on our chart,
     // we'll need to listen to the window resize event
     redrawCharts = function(){
         d3Selection.selectAll('.stacked-bar').remove();
 
         createStackedBarChartWithTooltip();
-        createHorizontalStackedBarChart();
+        // createHorizontalStackedBarChart();
```

Comment:
Do you want to keep this code in the demo?

Reply:
@Golodhros I will remove it! Just committed all the code by default